packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -27,16 +27,29 @@
     }
 
     &__title {
+      margin-bottom: $carbon--layout-05;
       @include carbon--type-style('expressive-heading-03');
+    }
 
-      margin-bottom: $carbon--layout-05;
+    &__title,
+    &__content {
+      color: $text-01;
     }
```

Comment:
Just a thought. Instead of spelling these classnames out fully down below it might be good to do the following.

```scss
&__title,
&__content {
  ...
  .#{$prefix}--card--inverse  & {
    ....
  }
}

Reply:
I thought for styling, carbon doesn't go more than two levels deep?

Reply:
Oh if that's what you're going for I suppose. I'm not a believer of that restriction I suppose. 😉